doc/sphinxman/source/scf.rst

```diff
@@ -739,6 +739,28 @@ To avoid this, either set |scf__df_basis_scf| to an auxiliary
 basis set defined for all atoms in the system, or set |scf__df_scf_guess|
 to false, which disables this acceleration entirely.
 
+Linear-Scaling Fock Builds
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Methods that build J and K matrices in linear time have been developed. Among these include
+the Continuous Fast Multipole Method (CFMM) method to build J matrices, described in [White:1994:8]_,
+as well as the Linear Exchange (LinK) Method to build K matrices, described in [Ochsenfeld:1998:1663]_.
+Currently, the CFMM method has not been implemented yet, but the LinK method is available to use as part
+of the Direct SCF algorithm. Linear-Scaling direct algorithms are a fast alternative to density
+fitting when the amount of available memory is low, especially when combined with Density-matrix based ERI
+screening and incremental Fock builds.
+
+To turn on and control the LinK algorithm, here are the list of options provided.
+
+DO_LINEAR_EXCHANGE
```

Comment:
Follow the style of the CPHF options mentioned on this page (`|cphf__solver_maxiter|: maximum number of iterations`) for these options.